llvm/lib/Target/X86/X86InstrFragments.td

```diff
@@ -266,7 +261,7 @@ def X86add_flag  : SDNode<"X86ISD::ADD",  SDTBinaryArithWithFlags,
 def X86sub_flag  : SDNode<"X86ISD::SUB",  SDTBinaryArithWithFlags>;
 def X86smul_flag : SDNode<"X86ISD::SMUL", SDTBinaryArithWithFlags,
                           [SDNPCommutative]>;
-def X86umul_flag : SDNode<"X86ISD::UMUL", SDT2ResultBinaryArithWithFlags,
+def X86umul_flag : SDNode<"X86ISD::UMUL", SDTBinaryArithWithFlags,
```

Comment:
Can we just delete this line? It's not used.

Reply:
I would rather keep it.  `UMUL` is created by `X86ISelLowering` and having a description in a td file would allow generating various info from it (a feature I'm currently working on but haven't created an RFC yet).